### DISCUSSION TITLE
Dashboard reload behavior

### DISCUSSION THREAD

**Discussion by Sebbl22** [datalust/seq-tickets#2063] [Help] [answered]
Hi,
we updated to the newest Seq Version 2023.4.10219 some weeks ago and noticed some behavior changes with dashboard reloading. I assume, it might be because of the custom time ranges and the tail feature in the dashboards.

_Note: The new URL options seem not to be documented. At least not here: https://docs.datalust.co/docs/dashboards (e.g. &tail is missing and it is unclear, if the &refresh parameter is still working, because the corresponding UI setting is not available anymore)._

We have Seq on a physical monitor in our office, which cycles through some webpages via the Firefox Extension [Tab Rotator](https://addons.mozilla.org/de/firefox/addon/tab-rotator/). Firefox 121.0.1 is used. The URL is fqdn/#/dashboards?dashboardId=dashboard-1007&range=8h&group=30m&tail&theme=dark-theme
As long as the Seq tab stays in the foreground, everything works as expected - that means the dashboard elements get "tailed" and refresh their data without a loading animation or any visible disruption. But when the Seq tab is in the background for approx. 20 sec, every dashboard element is completely reloaded.
![grafik](https://github.com/datalust/seq-tickets/assets/10319545/072bb7d6-d641-4077-b507-dcfc04a1be83)
Since our dashboard takes more than 15 seconds to fully load and we rotate every 25 seconds, we can only see the fully loaded dashboard for 10 seconds.
Is there any way to prevent this "full reload" behavior or maybe configure it? For this use case, it would be best if the tail-feature stays active in the background.

Thanks

**Comment by liammclennan** [comment]
Hi @Sebbl22 

Seq tries to preserve server resources by disabling queries when the UI is not visible. In your case, every time tab rotator chooses a different tab Seq suspends querying and hence has to reload the charts. 

I will investigate and see if there is a solution.

**Comment by liammclennan** [comment, marked as answer]
@Sebbl22 Firefox has some extensions that disable the page visibility API. Using one of those should mean that Seq keeps updating the charts, even when it is not visible.

**Comment by Sebbl22** [comment]
Thanks, we used one of those extensions and it works.

**Reply by liammclennan** [reply]
Great. Thanks for letting us know.
